superset-frontend/cypress-base/cypress/integration/dashboard/dashboard.helper.ts

```diff
@@ -110,8 +119,8 @@ export function drag(selector: string, content: string | number | RegExp) {
       (typeof target === 'string' ? cy.get(target) : target)
         .trigger('dragover', { dataTransfer })
         .trigger('drop', { dataTransfer })
-        .trigger('dragend', { dataTransfer })
-        .trigger('mouseup', { which: 1 });
+        .trigger('dragend', { dataTransfer, force: true })
+        .trigger('mouseup', { which: 1, force: true });
```

Comment:
this handles cases where dragging something onto a drop target causes the drop target to become covered by new elements